docs/computing/lumi-vs-mahti.md

```diff
@@ -0,0 +1,55 @@
+# How does LUMI-C differ from Mahti?
+
+[LUMI-C](https://docs.lumi-supercomputer.eu/computing/systems/lumic/) consists of 1536 AMD CPU nodes (2x64 cores each) with a performance of 5.6 petaflops. The node count of [Mahti](systems-mahti.md) is similar, 1404 AMD CPU nodes (2x64 cores each) corresponding to a performance of 7.5 Petaflops. Although the systems are very alike based on hardware, core count and performance, there are important differences related to access and use which are highlighted on this page.
```

Comment:
5.6 is linpack for Lumi, and 7.5 is theoretical for Mahti CPU nodes (1404 * 128 * 2.6 * 16). Theoretical for Lumi-C is 7.7 Petaflops (1536 * 128 * 2.45 * 16)

Reply:
And perhaps that Mahti has small amount of GPU nodes (24) and Lumi-C will be flanked by Lumi-D and Lumi-G with huge GPU capacity. 

Also, Mahti has 256GB memory in all CPU nodes, while Lumi-C in Puhti like fashion has also 512 and 1024 GiB nodes